<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GraphWid</class>
 <widget class="QWidget" name="GraphWid">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1185</width>
    <height>613</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" rowstretch="9,1" columnstretch="0,0">
   <item row="0" column="0" colspan="2">
    <widget class="QChartView" name="chartView" native="true">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <spacer name="horizontalSpacer">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>575</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,2,1,2">
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>选择更新方式</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <item>
        <property name="text">
         <string>即时更新</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>100ms</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>200ms</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>300ms</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>400ms</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>500ms</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>自定义时间</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="inputTime">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="placeholderText">
        <string>自定义/ms</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="inputBtn">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>确定</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QWidget</extends>
   <header>qchartview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
